client/src/components/Login/LoginForm.jsx

```diff
@@ -0,0 +1,73 @@
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+import React from 'react';
+
+// Creating schema
+const schema = Yup.object().shape({
+	email: Yup.string()
+		.required('Email is a required field')
+		.email('Invalid email format'),
+	password: Yup.string()
+		.required('Password is a required field')
+		.min(8, 'Password must be at least 8 characters'),
+});
+
+function LoginForm() {
+	return (
+		<>
+			{/* Wrapping form inside formik tag and passing our schema to validationSchema prop */}
+			<Formik
+				validationSchema={schema}
+				initialValues={{ email: '', password: '' }}
+				onSubmit={(values) => {
+					alert(JSON.stringify(values));
+				}}
+			>
+				{({
+					values,
+					errors,
+					touched,
+					handleChange,
+					handleBlur,
+					handleSubmit,
+				}) => (
+					<div className="login">
+						<div className="form">
+							<form noValidate onSubmit={handleSubmit}>
```

Comment:
Use the <Form /> component from formik instead of HTML form. It comes with onSubmit, handleSubmit, etc.
https://formik.org/docs/api/form